File: Python-164/chapter.10/lab10.10/offeredCourse.py
from course import Course

class OfferedCourse(Course):

    def __init__(self):
        self.instructorName = ""
        self.term = ""
        self.classTime = ""

    def printInfo(self):
        super().printInfo()
        print(f"   Instructor Name: {self.instructorName}\n   Term: {self.term}\n   Class Time: {self.classTime}")

File: Java-164/chapter.10/lab10.10/OfferedCourse.java
public class OfferedCourse extends Course {
   // TODO: Declare private fields - instructorName, term, classTime
    private String instructorName, term, classTime = "";

   // TODO: Define mutator methods -
   //      setInstructorName(), setTerm(), setClassTime()
    public void setInstructorName(String instructorName) {
        this.instructorName = instructorName;
    }

    public void setTerm(String term) {
        this.term = term;
    }

    public void setClassTime(String classTime) {
        this.classTime = classTime;
    }
   // TODO: Define accessor methods -
   //      getInstructorName(), getTerm(), getClassTime()
    public String getInstructorName() {
        return this.instructorName;
    }

    public String getTerm() {
        return this.term;
    }

    public String getClassTime() {
        return this.classTime;
    }
}